Cargo.toml

```diff
@@ -14,7 +14,7 @@ kvm-v4_20_0 = []
 fam-wrappers = ["vmm-sys-util"]
 
 [dependencies]
-vmm-sys-util = { version = ">=0.2.0", optional = true }
+vmm-sys-util = { version = ">=0.8.0", optional = true }
 
 versionize = { version = ">=0.1.4" }
```

Comment:
Should we bump this due to vmm-sys-util version upgrade ?

Reply:
done